src/lib.rs

```diff
@@ -224,6 +224,20 @@ impl Event {
         let msg = unsafe { std::slice::from_raw_parts(seq.buffer as *mut u8, seq.len as usize) };
         println!("fields: {:?}", std::str::from_utf8(msg).unwrap());
     }
+    ///This is a wrapper of
+    /// [tep_record_print_fields](https://www.trace-cmd.org/Documentation/libtraceevent/libtraceevent-field_print.html).
```

Comment:
This is missing a line of documentation/comment explaining what the function does. See line 211 for print_fields()